src/main/java/com/ironhack/ironLibrary/model/Student.java

```diff
@@ -11,16 +11,21 @@ public class Student {
 
     @Setter(AccessLevel.NONE)
     @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Integer usn;
+//    @GeneratedValue(strategy = GenerationType.UUID)
+    private String usn;
 
     private String name;
 
     public Student() {
 
     }
-    public Student(Integer usn, String name) {
+    public Student(String usn, String name) {

```

Comment:
Variable called usn of data type string, representing the Universal Student Number and acting as the unique identifier for the table student (Private member)